### PR TITLE
npm updates march

### PR DESCRIPTION
**Pull request by illepic**
Update all dependencies and resulting changes to linters/formatters.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "lint:scss": "stylelint '**/*.{css,scss}'",
     "lint:twig": "composer twig-lint lint ./apps && composer twig-lint lint ./source",
     "lint": "npm run lint:js; npm run lint:scss; npm run lint:twig",
-    "fmt:js": "npm run lint:js -- --fix && prettier --write '**/*.{js,json,vue}'",
+    "fmt:js": "npm run lint:js -- --fix",
```
Comment (jryanconklin) [inline package.json]: Did the use here change with a new version? Or is this a fix? It's broken currently on AWN and I waaant it.

File: source/default/_data/scssVariables.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
       "Roboto",
       "Helvetica Neue",
       "Arial",
+      "Noto Sans",
```
Comment (jryanconklin) [inline source/default/_data/scssVariables.json]: Just to be super clear, this is coming from the Bootstrap 4 update.

File: source/default/_patterns/00-protons/non-printing/_bootstrap-overrides.scss [hunk 3]
```diff
@@ -18,8 +18,8 @@ $card-border-width: rem-calc(1px);
 
 // Spacing
 $spacers: (
```
Comment (jryanconklin) [inline source/default/_patterns/00-protons/non-printing/_bootstrap-overrides.scss]: Non-blocking. A complete aside, I have some questions on how we should be using spacers on projects. I'll ping you tomorrow.

File: source/default/_patterns/02-molecules/vue-widget/__tests__/vue-clock.test.js [hunk 4]
```diff
@@ -20,14 +23,15 @@ describe('vue-clock.vue', () => {
...
+  it('updates the seconds hand', () => {
     const { getByTestId } = render(vueClock);
     const secondsHand = getByTestId('seconds-hand');
     const before = secondsHand.style.transform;
+
     setTimeout(() => {
       const after = secondsHand.style.transform;
       expect(before).not.toEqual(after);
-      done();
+      // done();
```
Comment (jryanconklin) [inline source/default/_patterns/02-molecules/vue-widget/__tests__/vue-clock.test.js]: I'm guessing the afterEach(cleanup) removes this?

File: package.json (hunk 1 above)
Comment (illepic) [reply package.json]: `eslint-plugin-prettier` runs prettier on `eslint --fix`. We've technically been running prettier TWICE against all our js/scss/css/vue this whole time!

File: source/default/_data/scssVariables.json (hunk 2 above)
Comment (illepic) [reply source/default/_data/scssVariables.json]: Indeed.

File: source/default/_patterns/00-protons/non-printing/_bootstrap-overrides.scss (hunk 3 above)
Comment (illepic) [reply source/default/_patterns/00-protons/non-printing/_bootstrap-overrides.scss]: Really want to see us using the definitions in Bootstrap if a Bootstrap project. `$spacers` is left over from the early days of pattern lab starter.

File: source/default/_patterns/02-molecules/vue-widget/__tests__/vue-clock.test.js (hunk 4 above)
Comment (illepic) [reply source/default/_patterns/02-molecules/vue-widget/__tests__/vue-clock.test.js]: The `useFakeTimers()` removes the need for this.

File: source/default/_patterns/00-protons/non-printing/_bootstrap-overrides.scss (hunk 3 above)
Comment (jryanconklin) [reply source/default/_patterns/00-protons/non-printing/_bootstrap-overrides.scss]: Awesome. I’ll need to check how to do this in Bootstrap. It’s inconsistent on my current project and I’d like to clear it up. I’ll reach out tomorrow.

**Pull request merged**